packages/lsp/src/plugin.ts

```diff
@@ -25,7 +29,8 @@ export type LogCommunication = boolean;
 /**
  * Language Server Protocol settings.
  */
-export interface LanguageServers {
+export interface LanguageServersExperimental {
```

Comment:
Is there a reason for placing `Experimental` in the interface?

This will force code change even if the final interface version is identical to the experimental one.

Reply:
The interfaces in this file are generated from the `plugin.json` schema file. Since I want the label of the LSP setting section to be `Language Servers (Experimental)`, I must set the title field of the schema to the same value. And this leads to the interface named `LanguageServersExperimental`.
 https://github.com/jupyterlab/jupyterlab/blob/db5cbee90856844fb623c57d9d13fadb479a9dc4/packages/lsp-extension/schema/plugin.json#L5

![lsp](https://user-images.githubusercontent.com/4451292/194378796-82f19148-83e4-4a82-97e4-3662a8564a90.png)

Reply:
Ok thanks